test/cli/commands_test.py

```diff
@@ -1220,35 +1263,36 @@ class TestProgressBars:
     """
 
     def test_cli_lint_disabled_progress_bar(
-        self, mock_disable_progress_bar: MagicMock
+        self, mock_disable_progress_bar: MagicMock, monkeypatch
     ) -> None:
         """When progress bar is disabled, nothing should be printed into output."""
+        monkeypatch.chdir("test/fixtures/")
         result = invoke_assert_code(
-            ret_code=65,
             args=[
                 lint,
                 [
                     "--disable_progress_bar",
-                    "test/fixtures/linter/passing.sql",
+                    "linter/passing.sql",
+                    "--dialect=ansi",
                 ],
             ],
         )
         raw_output = repr(result.output)
 
-        assert "\rpath test/fixtures/linter/passing.sql:" not in raw_output
+        assert "\rpath linter/passing.sql:" not in raw_output
         assert "\rparsing: 0it" not in raw_output
         assert "\r\rlint by rules:" not in raw_output
 
     def test_cli_lint_enabled_progress_bar(
-        self, mock_disable_progress_bar: MagicMock
+        self, mock_disable_progress_bar: MagicMock, monkeypatch
     ) -> None:
         """When progress bar is enabled, there should be some tracks in output."""
+        monkeypatch.chdir("test/fixtures/")
         result = invoke_assert_code(
-            ret_code=65,
```

Comment:
IIUC, checking for `exit_code=65` was because `passing.sql` was actually not passing the linter. I fixed `passing.sql`, so now the exit code is 0, indicating there were no linting issues.

Reply:
Should we assert ret_code=0 then?